tests/common/docker_image.py

```diff
@@ -265,7 +265,7 @@ def __init__(
     @property
     def tag(self) -> str:
         """Get the image tag."""
-        return "trufflesuite/ganache-cli:v6.12.1"
+        return "trufflesuite/ganache-cli:latest"
```

Comment:
@marcofavorito safe to do?

Reply:
ask Ganache developers :rofl: 
Joking. We often adopted the approach to fix all the versions of test dependencies as much as we can; I'd stick with `v6.12.1`, or the current latest. But happy to put `latest`, since I guess the core features would be backward-compatible

Reply:
Ok, I think let's try `latest`. The core features should be sufficiently stable.